### ISSUE TITLE
Messages go to all channels since update.

### ISSUE THREAD

**Issue by dmnchild** [kangarko/ChatControl-Pro#1141]
Since latest update yesterday, when anyone is joined to more than one channel the test they type goes into *all* channels instead of just the main originating one (which is global for me)

edit: Every time i login on admin account i rejoin all the channels i left prior so it doesnt seem to be saving my previous channels either (my work around.)

edit2: Even on config reload its forcing staff back into all channels.

**Comment by dmnchild** [comment]
Eh well, was a goodish run for a few days. back to venturechat. groups work and you can focus one as main (your choice) and /l /admin /whatever to chat in the others without broadcasting to all. Love all the extra features, but need basics to work to be viable. Would for sure come back when this was resolved.

**Comment by kangarko** [comment]
Hey, I sent you a PM, I think we can get this resolved.

Thanks

**Comment by NerdyBirder** [comment]
We have this problem currently on our server. /chc reload fixes the chat temporarily but then after a few minutes it goes back to sending the chat in all available channels for that player. For our admins its  8 channels so this is a huge problem right now.

We updated to Chatcontrol version 8.3.6 yesterday but that caused problems with channels not sending to all our bungeecord servers so I moved us back to 8.3.5 and used some files from a few weeks ago for formatting, channels, and others while using a new settings.yml. This change seemed to work then it started sending chats to all channels.


[ChatControl.zip](https://github.com/kangarko/ChatControl-Pro/files/3461188/ChatControl.zip)

**Comment by kangarko** [comment]
Hey, can you please set Debug to [channels-join] and provide me the console output when the plugin triggers all channels for a player?

As for updating to the latest CCP, you also need to update your BungeeControl.

**Comment by NerdyBirder** [comment]
Sorry for the delayed response. I was just reassured today that when this issue was happening we did update to the most recent BungeeControl as well as the most recent ChatControl version. We have since moved to a an older version of ChatControl that was working better and are not having this issue anymore. We are planning on trying out the new version of CC and BungeeControl soon and I will post updates here and do the debug instructions you posted above if we see this problem again.
